frontend/src/metabase/core/components/AccordionList/AccordionListCell.jsx

```diff
@@ -137,7 +137,7 @@ export const AccordionListCell = ({
       >
         <span
           className={cx(
-            "p1 flex-auto flex align-center",
+            "p1 flex align-center",
```

Comment:
Have you checked that there are no visual regressions because of this?

Reply:
Indeed there were was a regression - I explained how I fixed it here https://github.com/metabase/metabase/pull/31653#discussion_r1242786382